sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_models.py

```diff
@@ -89,8 +88,7 @@ def enabled(self):
     @property
     def not_before(self):
         # type: () -> datetime
-        """
-        Not-before time, in UTC
+        """Not-before time, in UTC
```

Comment:
Not really sure what a not-before time is. Guessing this is documented more thoroughly elsewhere?

Reply:
added more descriptive docstring, thanks!